src/webextension/list/manage/containers/all-items.css

```diff
@@ -2,20 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.all-items {
-  border-inline-end: 1px solid #d7d7db;
```

Comment:
We need this border to follow the visual spec, but it really belongs elsewhere. Specifically, it should go in `src/webextension/list/manage/components/app.css` in the definition for `.app-main > aside`. (This also fixes some wonkiness in the styles there, which is nice.)